microprofile-opentracing/src/main/java/org/jboss/eap/qe/microprofile/opentracing/v10/MyService.java

```diff
@@ -0,0 +1,20 @@
+package org.jboss.eap.qe.microprofile.opentracing.v10;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.opentracing.Traced;
+
+import io.opentracing.Tracer;
+
```

Comment:
Could you add some javadoc what is the purpose of this class, please?

Reply:
Fixed, thanks.